types/underscore/underscore-tests.ts

```diff
@@ -474,6 +474,45 @@ _(explicitNumberDictionary).each((value, key, collection) => {
     _.chain(anyValue).forEach(anyCollectionIterator); // // $ExpectType _Chain<any, any>
 }
 
+// get
+
+{
+    // tslint:disable-next-line:no-inferrable-types
+    const num: number = 2;
```

Comment:
Out of curiosity: why not just write `const num = 2`?

Reply:
`2` was getting on the type result, like `2 | something`.

Reply:
I was writing the values wrong. Remove the annotation.